Willow.podspec

```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
 
   s.source = { :git => "https://github.com/Nike-Inc/Willow.git", :tag => s.version }
   s.source_files = "Source/*.swift"
-  s.swift_version = "4.2"
+  s.swift_versions = ['4.2', '5.0']
```

Comment:
Can we switch over to using double instead of single quotes?

Reply:
Done!